src/App.php

```
@@ -46,11 +46,26 @@ public function run()
+    private function renderResponse($response)
+    {
+        if ($response === null) {
+            http_response_code(204); // No Content
+        } elseif (is_string($response)) {
+            echo $response; // Render string response
+        } elseif ($response instanceof Response) {
+            $response->send(); // Render custom Response object
+        } else {
+            header('Content-Type: application/json');
+            echo json_encode($response); // Fallback to JSON encoding
+        }
```

PHPMD (2.15.0)

The method renderResponse uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)